.coveralls.yml

```diff
@@ -0,0 +1 @@
+ repo_token: cnwBfq9loEWhbObghcFjjnl3IcB6d99Su
```

Comment:
🤔 This should probably be kept in a .env file.

Reply:
okay